specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-04-01-preview/ContentProductTemplates.json

```diff
@@ -0,0 +1,298 @@
+        },
+        "providers": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataProviders",
+          "description": "Providers for the content item"
+        },
+        "firstPublishDate": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataFirstPublishDate",
+          "description": "first publish date content item"
+        },
+        "lastPublishDate": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataLastPublishDate",
+          "description": "last publish date for the content item"
+        },
+        "customVersion": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataCustomVersion",
+          "description": "The custom version of the content. A optional free text"
+        },
+        "contentSchemaVersion": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataContentSchemaVersion",
+          "description": "Schema version of the content. Can be used to distinguish between different flow based on the schema version"
+        },
+        "icon": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataIcon",
+          "description": "the icon identifier. this id can later be fetched from the content metadata"
+        },
+        "threatAnalysisTactics": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataThreatAnalysisTactics",
+          "description": "the tactics the resource covers"
+        },
+        "threatAnalysisTechniques": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataThreatAnalysisTechniques",
+          "description": "the techniques the resource covers, these have to be aligned with the tactics being used"
+        },
+        "previewImages": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataPreviewImages",
+          "description": "preview image file names. These will be taken from the solution artifacts"
+        },
+        "previewImagesDark": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataPreviewImagesDark",
+          "description": "preview image file names. These will be taken from the solution artifacts. used for dark theme support"
+        },
+        "packageId": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataContentId",
+          "description": "the package Id contains this template"
+        },
+        "packageKind": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataPackageKind",
+          "description": "the packageKind of the package contains this template"
+        },
+        "packageName": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataDisplayName",
+          "description": "the name of the package contains this template"
+        },
+        "packagedContent": {
+          "type": "object",
+          "description": "the json to deploy"
+        }
```

Comment:
free form objects are not allowed , please use the correct schema for this

Reply:
Please use valid schema or a valid primitive type for this.